### DISCUSSION TITLE
cant install spotify

### DISCUSSION THREAD

**Discussion by naiyanrahman** [prasanthrangan/hyprdots#916] [Q&A] [answered]
I tried pacman also I tried via helper ( aur) and tried to build non of this work
![image](https://github.com/prasanthrangan/hyprdots/assets/77504180/b6fb2623-b0df-4b9d-aa75-3f73110a4c12)

**Comment by kRHYME7** [comment]
what is the output for ``` yay -S spotify ``` ?

**Reply by naiyanrahman** [reply]
the upper screenshort

**Reply by kRHYME7** [reply]
Try to run this first 

```
sudo pacman -Sy

sudo pacman -Sy archlinux-keyring


```

**Reply by naiyanrahman** [reply]
done,

**Reply by kRHYME7** [reply]
try to install it . 
```
yay  -Sy spotify
```

Remember to remove all your  build attempts of spotify

**Reply by naiyanrahman** [reply]
its works now and spotify installed successfully thanks

**Reply by kRHYME7** [reply, marked as answer]
Goodluck to your Linux journey! Please read the Archwiki why you got those GPG errors .


BTW if your intention is to have the spotify configs from this repo you also need this. 

```
yay  -S  spicetify-cli
```
